crates/store/re_types/src/blueprint/datatypes/latest_at_query_ext.rs

```diff
@@ -0,0 +1,12 @@
+use re_types_core::datatypes::{TimeInt, Utf8};
+
+use super::LatestAtQuery;
+
+impl Default for LatestAtQuery {
+    fn default() -> Self {
+        Self {
+            timeline: Utf8::from("log_time"),
+            time: TimeInt::MAX,
+        }
+    }
+}
```

Comment:
This default is neither useful, nor very robust (hardcoded timeline name), but required for reflection to work.